README.md

```diff
@@ -1 +1,62 @@
+# **BartaEngine**
+
+_This is a c++ library which implements core features of a physics engine. The core concepts for now are:_ 
+* _events handling system_
+* _velocity calculation_
+* _collision detection and collision responses_
+
+_The project is open to collaboration at my GitHub account as there are plenty of ideas for new functionalities. This program is a base for my Master's degree research about soft bodies simulation.A c++ library which implements core features of a physics engine. The core concepts for now are: events handling system, velocity calculation, collision detection and collision response. The project open to collaboration at my GitHub account as there are plenty of ideas for new functionalities. This program is a base for my Master's degree research about soft bodies simulation._
```

Comment:
`_The project is open to collaboration at my GitHub account as there are plenty of ideas for new functionalities. This program is a base for my Master's degree research about soft bodies simulation._`